synth.py

```diff
@@ -75,13 +75,26 @@
    "dev/src/v1beta1/firestore_client.ts",
    "/protos/protos'",
    "/protos/firestore_v1beta1_proto_api'"
- )
+)
 s.replace(
   "dev/test/gapic-firestore-v1beta1.ts",
   "/protos/protos'",
   "/protos/firestore_v1beta1_proto_api'"
 )
 
+# Mark v1beta1 as deprecated
```

Comment:
@AlanGasperini @alexander-fenster so this is a funny use for synthtool :) For cases where a specific API version itself is deprecated, how do we let customers know?

Reply:
@lukesneeringer Is there any annotation that might help mark the whole API version as deprecated?